src/Farmer/Deploy.fs

```diff
@@ -105,11 +106,17 @@ module Az =
             | WhatIf -> "what-if"
             | Validate -> "validate"
 
+    let private parametersJson (parameters : (string * string) list) = 
+        let dict = new Dictionary<string, {| value: string |}>()
+        for (a, b) in parameters do
+          dict.Add(a, {| value = b |});
+        (dict |> JsonSerializer.Serialize).Replace("\"", "\\\"")
+
     let private deployOrValidate (deploymentCommand:DeploymentCommand) resourceGroup deploymentName templateFilename parameters =
         let parametersArgument =
             match parameters with
             | [] -> ""
-            | parameters -> sprintf "--parameters %s" (parameters |> List.map(fun (a, b) -> $"%s{a}=%s{b}") |> String.concat " ")
```

Comment:
You could probably simplify this as we don't actually need JSON serialisation here, just quoting and quote-escaping.
`(fun (a, b) -> $"%s{a}=%s{b}")` can become
`(fun (a, b) -> $"%s{a}=\"%s{b.Replace("\"","\\\"")}\"")`

Reply:
Ahh yes, that makes it simpler. Updated thanks!